templates/CRM/MembershipExtras/Form/MembershipPeriod/PeriodRules.tpl

```diff
@@ -0,0 +1,95 @@
+<div class="region region-content">
```

Comment:
I think I'm liking even more having a 'Settings' path for settings forms, that way we can put there both period rules settings form and membership extras settings form. Not to be done now, we could refactor it later, but it would be cool if you can move this form to `templates/CRM/MembershipExtras/Form/Settings/PeriodRules.tpl` for `CRM/MembershipExtras/Form/Settings/PeriodRules.php`.